amplify/data/resource.ts

```diff
@@ -15,8 +17,16 @@ const schema = a
         email: a.string(),
         firstName: a.string(),
         lastName: a.string(),
+        totalEarnings: a.float(),
+        userId: a.id(),
+        sessionsAttended: a.belongsTo("SessionAttended", "userId"),
       })
       .authorization((allow) => [allow.owner()]),
+    SessionsAttended: a.model({
+      numberOfSessions: a.integer(),
+      earningsThatSession: a.float(),
+      date: a.hasMany("User", "userId"),
```

Comment:
You'll have to create a SessionsAttended ID field instead of making the userId the key used to relate the two fields